arm9/source/common/swkbd.h

```diff
@@ -87,3 +107,5 @@ enum {
 
 #define ShowKeyboardOrPrompt (TouchIsCalibrated() ? ShowKeyboard : ShowStringPrompt)
 bool PRINTF_ARGS(3) ShowKeyboard(char* inputstr, u32 max_size, const char *format, ...);
+bool BuildKeyboard(TouchBox* swkbd, const char* keys, const u8* layout, bool multi_line);
```

Comment:
This is exposed to prevent rebuilds between keypresses in the `MemTextViewer` function.

Reply:
A lot of these Github comments should probably be code comments instead

Reply:
@BlueRaja done. I went through and added code comments where I felt they were relevant. If you have any other concerns, please let me know. 😁